microhapulator/cli/filter.py

```diff
@@ -32,15 +33,15 @@ def subparser(subparsers):
         "--static",
         metavar="ST",
         type=int,
-        default=None,
+        default=5,
         help="global fixed read count threshold",
     )
     cli.add_argument(
         "-d",
         "--dynamic",
         metavar="DT",
         type=float,
-        default=None,
+        default=0.02,
```

Comment:
Current behavior is that all thresholds are disabled by default. Results are messy without thresholds—so messy, by my initial thinking, that the user would be forced to make deliberate decisions about the thresholds.

I decided to replace the disabled-by-default behavior with conservative defaults that we've been using.